### PR TITLE
JIRA: VZ-9710 add Kubernetes-OCNE version metadata to control plane provider

### PR DESCRIPTION
**Pull request by jmaron99**
The metadata is more appropriately associated to a provider since it is required for both VZ and non-VZ scenarios.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -66,7 +66,15 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} \
 # Production image
 FROM ghcr.io/oracle/oraclelinux:8-slim
 RUN microdnf update \
-    && microdnf clean all
+    && microdnf clean all \
```
Comment (mgianatagh) [inline Dockerfile:69]: should the "clean all" be the last statement executed for the RUN?

Comment (jmaron99) [reply Dockerfile:69]: Yeah - that makes more sense

File: go.mod [hunk 2]
```diff
@@ -21,8 +21,10 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.8.1
```
Comment (mgianatagh) [inline go.mod:24]: are any PLS approvals required?  I see we are using older versions of testify and zap in VPO.

Comment (jmaron99) [reply go.mod:24]: No clue.  The version of zap is unchanged - it was just moved by a required `go mod tidy` invocation.  Testify I can try to update to the same version

Comment (jmaron99) [reply go.mod:24]: Added the same replace directive for testify as is in VZ go.mod

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**